operations/helm/charts/mimir-distributed/CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ Entries should include a reference to the Pull Request that introduced the chang
   - Renamed helm config values `minio.accessKey` to `minio.rootUser`.
   - Renamed helm config values `minio.secretKey` to `minio.rootPassword`.
   - Minio container images are now loaded from quay.io instead of Docker Hub. Set `minio.image.repository` value to override the default behavior.
+* [CHANGE] Enable [query sharding](https://grafana.com/docs/mimir/latest/operators-guide/architecture/query-sharding/) by default. If you override the value of `mimir.config`, then take a look at `mimir.config` in the `values.yaml` from this version of the chart and incorporate the differences. If you override `mimir.config`, then consider switching to `mimir.structuredConfig`. #2655
```

Comment:
Let's mention that setting `mimir.structuredConfig.parallelize_shardable_queries` to `false` can be used to turn off query sharding.

Reply:
added and rebased onto the lastest main. PTAL